server/middleware/populateSystemClientToken.ts

```diff
@@ -0,0 +1,24 @@
+import { RequestHandler } from 'express'
+import logger from '../../logger'
+import { dataAccess } from '../data'
+
+export default function populateClientToken(): RequestHandler {
+  const { hmppsAuthClient } = dataAccess()
+
+  return async (req, res, next) => {
+    try {
+      if (res.locals.user) {
+        const systemClientToken = await hmppsAuthClient.getSystemClientToken(res.locals.user.username)
```

Comment:
Are we getting a system client token on every UI request?

Reply:
Yes, it is added in the app.ts  `app.use(populateClientToken())`
Maybe we can add it at when user login successfully only.

Reply:
We need a ticket to get this on demand, and cache / re-use from Redis .. I'll create one.